timeseries/src/autogluon/timeseries/models/gluonts/mx/models.py

```diff
@@ -434,7 +447,7 @@ def _get_estimator(self):
             return self.gluonts_estimator_class.from_hyperparameters(**self._get_estimator_init_args())
 
 
-class GenericGluonTSModelFactory(AbstractTimeSeriesModelFactory):
+class GenericGluonTSMXNetModelFactory(AbstractTimeSeriesModelFactory):
```

Comment:
Do we actually ever use this class in the code? Or is this just functionality for advanced users to wrap arbitrary GluonTS models in AGTS?

Reply:
That was the intention, but we're not really promoting it anywhere.